scalyr

```diff
@@ -538,10 +538,8 @@ def commandFacetQuery(parser):
 # Implement the "scalyr timeseries-query" command.
 def commandTimeseriesQuery(parser):
     # Parse the command-line arguments.
-    parser.add_argument('--timeseries', default='', 
-                        help='ID of the timeseries to query')
-    parser.add_argument('filter', nargs='*', default='',
-                        help='search term or filter expression, used only if --timeseries is not provided')
+    parser.add_argument('filter', nargs=1, default='',
+                        help='search term or filter expression')
```

Comment:
`nargs='*'` was a mistake all along, that you've corrected w/ this change?

Reply:
It used to be optional (b/c it wasn't required if `--timeseries` was provided); this change is just a cut-and-paste from the numeric-query command (whose params are now identical).